app/models/concerns/vacancy_pay_package_validations.rb

```diff
@@ -0,0 +1,17 @@
+module VacancyPayPackageValidations
+  extend ActiveSupport::Concern
+  include ApplicationHelper
+
+  included do
+    validates :salary, presence: true
+    validates :salary, length: { minimum: 1, maximum: 256 }, if: :salary?
+  end
+
+  def salary=(value)
+    super(sanitize(value, tags: []))
```

Comment:
Why are you passing an array with no tags here?

Reply:
Because no tags are allowed